frontend/www/js/omegaup/components/problem/Collection.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <h1 class="card-title">{{ T.collectionTitle }}</h1>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { omegaup } from '../../omegaup';
+import T from '../../lang';
+import { types } from '../../api_types';
+import * as ui from '../../ui';
```

Comment:
nit:

```suggestion
import { Vue, Component } from 'vue-property-decorator';
import T from '../../lang';
```

esto no se está usando.